apps/web/pages/api/teams/api/create.ts

```diff
@@ -0,0 +1,132 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
Self review: This endpoint is the success_url of Stripe checkout session, so after payment user is re-directed here, where "AwaitingPaymentTeam" is fetched using awaiting payment team id in the metadata and actual Team is created and returned to user.